js/promisehelpers.js

```diff
@@ -0,0 +1,43 @@
+/*jshint esversion: 6 */
+//Function object for exporting
+function RestaurantLogging() {}
+/*
+ * Function to deal with chain of promises and their logging
+ * @param callbacks- functions to perform
+ * @return resolve/reject - either data or rejection reason
+ */
+RestaurantLogging.promiseAllLogging = function(callbacks) {
+    return new Promise(function(resolve, reject) {
+        Promise.all(callbacks).then(result => {
+                console.log(result);
```

Comment:
This request can't be fullfilled until I deal with actual logging. Right now this logging is neccesary.